### PR TITLE
Update SQS policy to allow ICA to send notifications.

### PR DESCRIPTION
**Pull request by reisingerf**
Add sns:SendMessage permissions to incoming queue
Update tests to check for additional permissions
Refactor construct directory name to constructs

### CODE REVIEW

**Comment by victorskl** [comment]
Hehe. Lint failed. 😁

File: test/stateful/ica-event-pipe/icaEventPipeConstruct.test.ts [hunk 1]
```diff
@@ -38,22 +41,47 @@ describe('IcaEventPipeConstruct', () => {
...
+              Principal: {
+                AWS: {
+                  'Fn::Join': ['', ['arn:', Match.anyValue(), ':iam::' + icaTestAccount + ':root']],
+                },
+              },
+            }),
+          ]),
+        },
+      })
+    );
```
Comment (victorskl) [inline test/stateful/ica-event-pipe/icaEventPipeConstruct.test.ts:84]: Auuu, Nice! 👍 Very comprehensively testing.

**Review by victorskl — APPROVED** [review]
LGTM!

File: test/stateful/ica-event-pipe/icaEventPipeConstruct.test.ts (hunk 1 above)
Comment (reisingerf) [reply test/stateful/ica-event-pipe/icaEventPipeConstruct.test.ts:83]: That was where I spent most of the time ;-)

**Comment by reisingerf** [comment]
Argh, linting... my nemesis!
Try fix on the go...

**Comment by reisingerf** [comment]
Ha! GitHub mobile to the rescue ;-)

**Pull request merged**